concept/type/relationtype.feature

```diff
@@ -206,13 +202,36 @@ Feature: Concept Relation Type and Role Type
     Then relation(parentship) get role(parent) is abstract: true
     Then relation(parentship) get role(child) is abstract: true
     When transaction commits
-    When session opens transaction of type: write
-    Then relation(parentship) create new instance; throws exception
     When session opens transaction of type: read
     Then relation(parentship) is abstract: true
     Then relation(parentship) get role(parent) is abstract: true
     Then relation(parentship) get role(child) is abstract: true
-    Then relation(parentship) create new instance; throws exception
+
+  Scenario: relation and role types can be set to abstract when a subtype has instances
```

Comment:
split a new scenario for just testing setting a type as abstract when it has indirect instances